lms/djangoapps/student_account/harambee.py

```diff
@@ -87,8 +85,8 @@ def auth_complete_params(self, state=None):
     def get_user_details(self, response):
         data = jwt.decode(response.get('id_token'), verify=False)
         return {'username': data.get('CandidateGUID')[0:-6],
-                'email': "{g}@harambeecloud.com".format(g=data.get('CandidateGUID')),
-                'fullname': "{f} {l}".format(f=data.get('Firstname'), l=data.get('Lastname')),
+                'email': u"{g}@harambeecloud.com".format(g=data.get('CandidateGUID')),

```

Comment:
this is also not part of the reported story. Please remove from the PR

Reply:
same here. this is a code for "fix for UnicodeDecode error for instructor API" bugfix

Reply:
@nerfani see my previous comment. Thanks